gen/enum.go

```diff
@@ -110,19 +114,27 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 					return nil
 			<end ->
 				default:
-					<$val>, err := <$strconv>.ParseInt(<$s>, 10, 32)
+					<if checkErrOnUnknownEnum ->
+						return <$fmt>.Errorf("unknown enum value %q for %q", <$s>, "<$enumName>")
+					<else ->
+					<$val>, err := strconv.ParseInt(<$s>, 10, 32)
```

Comment:
If we're importing strconv with `<$strconv := import "..">`, we should use `<$strconv>` here to reference the package. The variable accounts for conflicting imports.
```suggestion
					<$val>, err := <$strconv>.ParseInt(<$s>, 10, 32)
```